bin/git-rename-remote

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+old=$1
+new=$2
+
+test -z $old && echo "old remote name required." 1>&2 && exit 1
+test -z $new && echo "new remote name required." 1>&2 && exit 1
+
+if git config --get "remote.$new.fetch"; then
+  git remote remove $new
+fi
+git remote rename $old $new
+git remote -v
```

Comment:
Maybe we should skip the `git remote -v` when `git remote rename` failed?

Reply:
such good advice, I've `set -e` this script.